### DISCUSSION TITLE
Tree Node Drag & Drop

### DISCUSSION THREAD

**Discussion by Arnon-E** [primefaces/community#1883] [PrimeReact] [answered]
I faced an issue when the Tree items became draggable, the keyboard keys were not working well (up, down, left, right, space, enter)
Can anyone no why? is it something that I'm doing wrong? I see it's either not working in the website example (https://primereact.org/tree/#dragdrop)

**Comment by nitrogenous** [comment, marked as answer]
Thank you for bringing this to our attention. It appears that this issue occurred during the accessibility implementation. We have noted your bug report and successfully reproduced the issue. A task will be created to address this, and we will work to resolve it as soon as possible.

Edit:
Issue Link: https://github.com/primefaces/primereact/issues/6631
